packages/fxa-settings/src/lib/utilities.ts

```diff
@@ -3,6 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import { AttachedClient } from '../models/Account';
+import { useLocation } from '@reach/router';
```

Comment:
Q: Is this import used somewhere, or somehow needed for `getSearchParams()` below?
Or could we use it for a default value for `locationHref` so we don't need to explicitly pass in `useLocation().href`

Reply:
This was leftover from something I refactored, thanks for catching.